doc/lexer-modes/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "lexer-modes"
+version = "0.1.0"
+edition = "2021"
+authors = ["Neal H. Walfield <neal@sequoia-pgp.org>"]
+
+[build-dependencies]
+lalrpop = { version = "0.20.2", path = "../../lalrpop" }
```

Comment:
The other examples have the same issue:

```
$ grep 'lalrpop =' $(find -name Cargo.toml)
./calculator/Cargo.toml:lalrpop = { version = "0.20.2", path = "../../lalrpop" }
./pascal/lalrpop/Cargo.toml:lalrpop = { version = "0.20.2", path = "../../../lalrpop" }
./nobol/Cargo.toml:lalrpop = { version = "0.20.2", path = "../../lalrpop" }
./lexer-modes/Cargo.toml:lalrpop = { version = "0.20.2", path = "../../lalrpop" }
./lexer/Cargo.toml:lalrpop = { version = "0.20.2", path = "../../lalrpop" }
```

Reply:
My point is that [an open PR](https://github.com/lalrpop/lalrpop/pull/903/files) bumps these numbers for all the other examples.  Whichever of this PR or #903 merges second will need to update to sync up, since #903 obviously doesn't currently bump them for this.